File: frontend/src/app/features/report/models/report.dto.ts
import {UserDto} from "../../user/models/user.dto";
import {FileDto} from "../../../core/models/file.dto";

export interface ReportDto {
    reportId: number;
    user: UserDto;
    file: FileDto;
    reportName: string;
    reportType: string;
    reportStartDate: Date;
    reportEndDate: Date;
    txDate: Date;
}

File: backend/src/main/kotlin/bo/edu/umsa/backend/dto/ReportDto.kt
package bo.edu.umsa.backend.dto

import java.sql.Timestamp

data class ReportDto(
    val reportId: Int,
    val user: UserPartialDto,
    val file: FilePartialDto,
    val reportName: String,
    val reportType: String,
    val reportStartDate: Timestamp,
    val reportEndDate: Timestamp,
    val txDate: Timestamp,
)
